socorro/external/crashstorage_base.py

```diff
@@ -179,14 +178,6 @@ def close(self):
         """Close resources used by this crashstorage."""
         pass
 
-    def is_mutator(self):
-        """Whether this storage class mutates the crash or not
-
-        By default, crash storage classes don't mutate the crash.
-
-        """
-        return False
-
```

Comment:
The Elasticsearch crashstorage was the only one that mutated the raw and processed crash data. Because we're changing it so that Elasticsearch crashstorage now creates new data structures, we don't need this mutator stuff anymore.